src/scaladoc/scala/tools/nsc/doc/Settings.scala

```diff
@@ -253,7 +253,7 @@ class Settings(error: String => Unit, val printMsg: String => Unit = println(_))
     }
   }
 
-  def appendIndex(url: String): String = url.stripSuffix("index.html").stripSuffix("/") + "/index.html"
+  def appendIndex(url: String): String = url.stripSuffix("index.html").stripSuffix("/") + "/"
```

Comment:
This is probably more of a naming problem as `appendIndex` is only used in `extUrlMapping` which is only used to get the base URL to which the file name is appended. This means that we shouldn't be linking to dirs directly.